packages/shared/models/board.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from 'mongoose'
+
+const boardSchema = new mongoose.Schema({
+  id: String,
+  title: String,
+  description: String,
+  createdAt: String,
+  admin: {
```

Comment:
admin is a user so we don't have to add separate field for that. What when there will be 2 admins? Remove